packages/drawer/src/Drawer/Drawer.styles.ts

```diff
@@ -21,48 +24,37 @@ const getBaseStyles = ({ theme }: { theme: Theme }) => css`
   width: 100%;
   max-width: ${PANEL_WIDTH}px;
   height: 100%;
-  overflow: auto;
 
   @media only screen and (max-width: ${MOBILE_BREAKPOINT}px) {
     max-width: 100%;
     height: 50vh;
   }
 `;
 
-const getOverlayOpenStyles = (theme: Theme) => css`
-  box-shadow: ${theme === Theme.Light
-    ? `-10px 0 10px -10px rgba(0, 0, 0, 0.3)`
-    : 'initial'};
+const overlayOpenStyles = css`
   opacity: 1;
   transform: none;
 
   @media only screen and (max-width: ${MOBILE_BREAKPOINT}px) {
```

Comment:
nit: maybe you can turn this into a reusable function so you don't have to keep repeating it?